src/SitemapIndex.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace Spatie\Sitemap;
+
+use Spatie\Sitemap\Tags\Tag;
+use Spatie\Sitemap\Tags\Sitemap;
+
+class SitemapIndex
+{
+    /** @var array */
+    protected $tags = [];
+
+    /**
+     * @return static
+     */
+    public static function create()
+    {
+        return new static();
+    }
+
+    /**
+     * @param string|\Spatie\Sitemap\Tags\Tag $tag
+     *
+     * @return $this
+     */
+    public function add($tag)
```

Comment:
I think you are right, I called it `$tag` for consistency with the `Sitemap\Sitemap` class